terraform/root_main.tf

```diff
@@ -22,3 +22,18 @@ module "dynamodb_kms_key" {
     service_names = ["cloudwatch"]
   }
 }
+
+module "reference_generator_lambda" {
+  source        = "./da-terraform-modules/lambda"
+  function_name = local.reference_generator_function_name
+  handler       = "uk.gov.nationalarchives.referencegenerator.Lambda::handleRequest"
+  policies = {
+    "ReferenceGeneratorLambdaPolicy${title(local.hosting_environment)}" = templatefile("${path.module}/templates/lambda/reference_generator_policy.json.tpl", {
```

Comment:
The policy name should be prefixed with `DA`. This can be derived from the `project` variable using the "upper" function: https://developer.hashicorp.com/terraform/language/functions/upper